js/notifications.js

```diff
@@ -40,7 +40,7 @@ Fliplet.Widget.register('PushNotifications', function () {
     var appPages = Fliplet.Env.get('appPages');
 
     if (Array.isArray(appPages) && appPages.length) {
-      var page = appPages.filter(function(page) { return page.id === parseInt(data.page);});
+      var page = appPages.filter(function(page) { return page.masterPageId === parseInt(data.page) || page.id === parseInt(data.page);});
```

Comment:
Line's getting a bit long...

Also, add `, 10` to `parseInt()`

```suggestion
      var page = appPages.filter(function(page) {
        return page.masterPageId === parseInt(data.page, 10) || page.id === parseInt(data.page, 10);
      });
```